assets/js/validation.js

```diff
@@ -1,6 +1,32 @@
 const axios = require('axios');
 
 let name = document.getElementById('name');
+let nameValidationResult = document.getElementById('validation-result-name');
+
+let team = document.getElementById('team');
+let teamValidationResult = document.getElementById('validation-result-team');
+
+function validate (input, result) {
+    result.innerText = '...';
+    axios.post(result.dataset.path, {input: input.value})
+        .then(function(response) {
+            if (response.data.valid) {
+                result.innerHTML = ":)";
+            } else {
+                result.innerHTML = ":(";
+            }
+        })
+        .catch(function (error) {
+            result.innerText = 'Error: ' + error;
+        });
+}
+
+name.onkeyup = () => validate(name, nameValidationResult);
+name.onchange = () => validate(name, nameValidationResult);
+
+team.onkeyup = () => validate(team, teamValidationResult);
+team.onchange = () => validate(team, teamValidationResult);
+
 let validationResult = document.getElementById('validation-result');
```

Comment:
Jei jau keiti kodą, tai reikia jį keisti **visą**. (`'validation-result'` :vs: `'validation-result-name')`)

Nes realiame darbe negalima bus pridedant naują funkcionalumą sugadinti senąjį:
![image](https://user-images.githubusercontent.com/1453957/70867714-fe3e6b80-1f80-11ea-9fbe-73c486dc1b9c.png)
Tai būtų laikoma grubi klaida, nes sukuriama **žala** klientui.

Tiek JavaScript pagrindų jau tikimasi iš kiekvieno.

Reply:
Tas pats matoma ir iš testų:
![image](https://user-images.githubusercontent.com/1453957/70867746-7a38b380-1f81-11ea-9d42-a32fcaa51bee.png)

Reply:
sveikas, labai atsiprasau, cia biski gavos nesusipratimas, nes man viskas tikrai gerai veike, bet kai parasei:

Atsinaujink versiją:

git remote add upstream https://github.com/aurelijusb/kickstart.git
git fetch upstream
git merge upstream/homework-2019-11-14,

man ten gavosi merge confliktas, ir neistryniau iki galo visu eiluciu dabar ziuriu, del to ten sudublikuotas kodas yra. o pas save viskas tvarkoj man rode nes pamirsau webpacka recompilint. tai tiek. as dabar cia ir taip ir taip galvoju, kaip taip galejo buti, nes sypsenos tai tikrai man veike pries ta upstream. Sorry uz apmaudzia klaida, dabar nutryniau ta dublikata, persikompilinau ir viskas veikia.